app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java

```diff
@@ -783,6 +785,9 @@ private void exitResizeMode(ResizeAction aResizeAction) {
             TelemetryWrapper.windowsResizeEvent();
             GleanMetricsService.windowsResizeEvent();
         }
+
+        mAttachedWindow.mWidgetPlacement.tintColor = 0xFFFFFFFF;
```

Comment:
I wonder if we should change all these hard code hex colors to constants:
```
final int WhiteTint = 0xFFFFFFFF;
final int GrayTint = 0x555555FF;
```

Reply:
Refactored hex colors to `Windows`